Add `total_count` GraphQL query & Remove unsafe block in `write_run_tcpdump` (#795)
aicers/giganto: CHANGELOG.md

    know which source the event is from.
- Changed the sorting of event search results for the all source from `cursor`,
  `source` to `timestamp`, `source`.
- Removed `unsafe` block in `write_run_tcpdump` while creating a temorary file.

> - Fixed `events_in_cluster` macro.

Author: done

>   - Added `into` expression to also support `total_count` query.

Reviewer: ```suggestion
  - Added `filter_into` expression to also support `total_count` query.
```

>   - Added `filter_into` expression to also support `total_count` query.

Reviewer: 여기도 마찬가지라고 생각하고 있습니
. 
```suggestion
  - Added `filter_into` expression to also support `totalCount` GraphQL query.
```

  - Added macro pattern to return the sum of all source request results.

### Added

- Added a `NetworkOptFilter` structure to allow network event search queries
  to request all sources.
- Added `handle_paged_events_for_all_source`, `get_peekable_iter_for_all_source`,
  `load_connection_for_all_source` functions for all source search.

> - Added `total_count` GraphQL query to return the number of events for a specific

Reviewer: 늦게 발견해서 죄송합니다만 Grapqhl API 중 query 종류를 의미하는 GraphQL query 는 camelCase 를 사용하므로 아래처럼 되어야할 것 같습니다.

```suggestion
- Added `totalCount` GraphQL query to return the number of events for a specific
```

Author: changelog 에서 쿼리를 추가에 대한 내용 작성시, giganto 내부에 쿼리를 구현한 함수 이름으로 작성하고 있어서 그대로 가는게 맞는 거같습니다.

Reviewer: 다른 리포지토리로의 링크이긴 하지만, https://github.com/aicers/review-web/pull/129#discussion_r1527455042 이러한 코멘트를 고려할 때, giganto도 마찬가지로 나아가야 하지 않을까 싶습니다.

Reviewer: ```suggestion
- Added `total_count` GraphQL query method to return the number of events for a specific
```

이러한 방향도 가능한 방향일 것 같아요.

Author: @sophie-cluml 링크주신 코멘트를 보니 기존 changelog의 Graphql query에 대한 부분도 다 변경하는게 맞을 것 같네요. 그렇다면 일단 현재 상태로 merge를 하고 Graphql query 표시 이슈를 별도로 생성해서 main,tis를 수정하도록 진행하는건 어떨까요?

Reviewer: 네 별도 이슈 생성 / 현재 PR 수정 둘 중 하나는 되어야 할 것 같아서, 별도 이슈 생성 부탁드리겠습니다.

Author: @sophie-cluml https://github.com/aicers/giganto/issues/805 이슈 생성 완료했습니다.
